### ISSUE TITLE
Server processing compression time is too long

### ISSUE THREAD

**Issue by kayw-geek** [skeeks-semenov/yii2-assets-auto-compress#51]
Compress assets  207.7 ms、Compress css code 72.6 ms、
![image](https://user-images.githubusercontent.com/29700073/54895790-42be3e80-4efc-11e9-87e7-f079ce74953b.png)
Why is that?

**Comment by skeeks-semenov** [comment]
@weikaiiii 
This happens once, with subsequent page updates, the finished file should already be taken.
Is your project online? Can I link to it?

**Comment by weikaiiii** [comment]
> @weikaiiii
> 这种情况发生一次，随后的页面更新，应该已经完成​​了已完成的文件。
> 你的项目在线吗？我可以链接到它吗？
Yes, the problem persists. The project address is [https://www.chasing-innovation.com](url)

**Comment by weikaiiii** [comment]
![image](https://user-images.githubusercontent.com/29700073/54901727-507ebe80-4f12-11e9-893b-985a4972faed.png)
It seems that the code has been written repeatedly.

**Comment by skeeks-semenov** [comment]
@weikaiiii 
Judging by the screenshot, yes)
I will check soon, thanks.

**Comment by weikaiiii** [comment]
> @weikaiiii
> Judging by the screenshot, yes)
> I will check soon, thanks.
You're welcome. I find that the _processingCss function in the local environment is slower than in the formal environment. Is it related to server configuration or PHP configuration?

**Comment by skeeks-semenov** [comment]
I corrected the duplication of logic, removed unnecessary.

_processingCss - in fact, uses this library to compile css code https://github.com/natxet/CssMin

If there are other libraries in mind that can do it on the fly faster. So we can try to introduce them. I have not dealt with this issue for a long time, so I don’t know, maybe there is something newer and better.

In my css projects in the html code is not very big, so I did not attach much importance to this. A css file compilation occurs once and then cached.

Although css in your project does not look great.
